x-pack/plugins/alerting/server/plugin.ts

```diff
@@ -274,7 +277,12 @@ export class AlertingPlugin {
     // Routes
     const router = core.http.createRouter<AlertingRequestHandlerContext>();
     // Register routes
-    defineRoutes(router, this.licenseState, plugins.encryptedSavedObjects);
+    defineRoutes({
```

Comment:
Changed this interface to options object instead of list of params bc the list is getting long and I'm planning to add logger as well in another PR